src/Components/DrumPad/index.js

```diff
@@ -34,7 +34,7 @@ const DrumPad = ({ count, layout, swapBeat }) => {
               <td
                 key={beatNum}
                 onClick={() => {
-                  console.log({ rowNum, beatNum });
+                  //console.log({ rowNum, beatNum, hasBeat });
```

Comment:
Instead of commenting this out, I suggest we should completely delete this line :)